src/types/dialog.ts

```diff
@@ -24,11 +28,15 @@ export type DialogHeaderProps = {
 export type DialogBodyProps = {
     children: ReactNode;
     maxHeight?: number | string;
```

Comment:
Is there a reason why we accept `maxHeight` as `number | string`, then to use it like `style={{maxHeight}}`. Should we maybe force user to use some predefined sizes, by strongly typing this property.

Reply:
Not yet. For now it will be flexible as we would need a designer to decide the predefined sizes we should have in the platform.